src/test/ui/consts/const-eval/ice-generic-assoc-const.rs

```diff
@@ -14,3 +13,6 @@ impl<T> Nullable for *const T {
         *self == Self::NULL
     }
 }
+
+fn main() {
+}
```

Comment:
No please do *not* change this test! Your PR is reverting the test back to the broken state where it doesn't actually trigger the ICE.